### DISCUSSION TITLE
Will Rspack have a stable release next year?

### DISCUSSION THREAD

**Discussion by AsuraKev** [module-federation/module-federation-examples#3158] [Q&A] [answered]
Hello! I am currently trialing out Modern js to explore some options around micro frontend. Garfish is currently the only option to support microfrontend, but since Module federation will be built into rspack so I am wondering when rspack will possibly come out of experimental?

Will new module federation be looking like garfish in terms of configuration schema or nah?

thanks :)

**Comment by AsuraKev** [comment]
@ScriptedAlchemy ^ :)

**Reply by hardfist** [reply, marked as answer]
yeah we will release 1.0 next year

**Reply by ScriptedAlchemy** [reply]
More than likely, yes. Rspack has covered a ton of ground already

**Reply by AsuraKev** [reply]
thanks guys, how different in terms of the usage of module federation 1.1 compare to Garfish and also will Modern.js framework adopt Module federation 1.1 as it comes out?  Currently Garfish is slightly less flexible in terms of some usage ~
